src/datapilot/utils/utils.py

```diff
@@ -40,3 +42,49 @@ def get_dir_path(path: str) -> str:
     :return:
     """
     return Path(path).parent
+
+
+# Will need to change this
+base_url = "http://localhost:5001"
+
+
+def upload_content_to_signed_url(file_path, signed_url):
+    api_client = APIClient()
+
+    with Path(file_path).open("rb") as file:
+        file_content = file.read()
+
+    return api_client.put(signed_url, data=file_content)
+
+
+def onboard_manifest(api_token, tenant, dbt_core_integration_id, manifest_path):
+    api_client = APIClient(api_token, base_url, tenant)
+
+    endpoint = "/dbt/v1/signed_url"
+    params = {"dbt_core_integration_id": dbt_core_integration_id, "file_type": "manifest"}
+    signed_url_data = api_client.get(endpoint, params=params)
+
+    if signed_url_data:
+        signed_url = signed_url_data.get("url")
+        file_id = signed_url_data.get("dbt_core_integration_file_id")
+        print(f"Received signed URL: {signed_url}")
+        print(f"Received File ID: {file_id}")
+
+        upload_response = upload_content_to_signed_url(manifest_path, signed_url)
+
+        if upload_response:
+            endpoint = "/dbt/v1/verify_upload"
```

Comment:
We should move this to the API client. I can see us using the same  endpoints for manifest catalog and everything else. So there will a common code which takes care of everything.